### ISSUE TITLE
Domain Tracking v2.x

### ISSUE THREAD

**Issue by uhlhosting** [postalserver/postal#1727]
## To Reproduce

Upgrading from v1.x to v2.x stops domain tracking while no documentation in place to guide towards fixing it, since Caddy got over Nginx, this is in our opinion something extra to handle on server level. Since we anyhow proxy it with nginx.

## Expected behaviour

A clear and concise description on what users need to keep count when upgrading from 1 to 2 and properly documented steps to fix the domain tracking.

## Environment details 

 - OS: Linux
 - Browser Safari
 - Version latest
 - Type deesktop

**Comment by willpower232** [comment]
This is already covered in the upgrade documentation

https://docs.postalserver.io/install/upgrade-to-v2#configuring-web-proxy-for-openclick-tracking

**Comment by uhlhosting** [comment]
@willpower232 would be glad to use it if would also work. And since you changed from nginx to caddy this surely made it easier for some and harder for others. Are so many missing points not beying documented. Not examples of configs not for caddy nor for nginx. So this surely is dovumented maybe for the internal team now not everyone works on this from inside. Its good to be usable for all. I have tried several configs. I preffered to use an extra ip to avoid all this confusion since its not working out of the box.
